### PR TITLE
Add connection string to bind info

### PR DESCRIPTION
**Pull request by stefanschneider**
This will add an a new member (connectionString) to the bind info that will could be used by apps or buildpack.

This update also adds some additions to the readme file.

### CODE REVIEW

**Comment by stefanschneider** [comment]
@mihaibuzgau @viovanov  can you please check out this PR

**Comment by mihaibuzgau** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -40,33 +42,36 @@ choco install mssqlserver2012express
...
 
-The brokerGoSqlDriver and brokerMssqlConnection are settings that the broker uses to connect to the mssql instance. brokerGoSqlDriver can be odbc (recommanded https://code.google.com/p/odbc/) for mssql (experimental https://github.com/denisenkom/go-mssqldb). brokerMssqlConnection is a map that is 
-converted into a connection string (e.g. "server=localhost;port=1433") consumed by odbc or mssql go library.
-Example for a local trusted brokerMssqlConnection with odbc driver:
+`logLevel` will set the logging level. Accepted levels: "debug", "info", "error", and "fatal".
+
+The `brokerGoSqlDriver` and `brokerMssqlConnection` are settings that the broker uses to connect to the mssql instance. `brokerGoSqlDriver` can be "odbc"" (recommanded https://code.google.com/p/odbc/) or "mssql" (experimental https://github.com/denisenkom/go-mssqldb). `brokerMssqlConnection` is a key-value json object that is
```
Comment (viovanov) [inline README.md]: typos
recommanded -> recommended

File: README.md [hunk 2]
```diff
@@ -40,33 +42,36 @@ choco install mssqlserver2012express
...
 		"driver": 	"sql server",
 		"trusted_connection": "yes"
 	}
+	
+`listeningAddr` and `brokerCredentials` are used for the brokers http server. The CF CloudController will use this setting to connect to the broker.
 
-listeningAddr and brokerCredentials are used for the brokers http server. The CF CloudController will use this setting to connect to the broker.
-
-dbIdentifierPrefix appended at the begining of the instance ID for the SQL Server database name, and it is appended at the begining of the binding id for the SQL Server user name. This will allow an adim to easily idetify the databases managed by a particular mssql broker.
+`dbIdentifierPrefix` is a string that is appended at the begining of the instance ID for the SQL Server database name, and at the begining of the binding id for the SQL Server user name. This will allow operators to easily idetify the databases managed by a particular mssql broker. Do not change this value on a existing mssql broker with active instances.
```
Comment (viovanov) [inline README.md]: typos
begining -> beginning
idetify -> identify
on a existing mssql broker -> on an existing mssql broker

File: README.md [hunk 3]
```diff
@@ -40,33 +42,36 @@ choco install mssqlserver2012express
...
 ## Configuration
 
-cf_mssql_broker_config.json is the default configuration file. The config file can be overridden with
-the following flag: -config=/new/path/config.json
+cf_mssql_broker_config.json is the default configuration file. The config file can be overridden with the following flag: -config=/new/path/config.json
 
-The servedMssqlBindingHostname and servedMssqlBindingPort need to be changed for every installetion.
-They are the hostname and port that are sent to the CF applicatoins, and need to be accessible from the CF application network. NOTE: Do not change this value on a existing mssql broker with active bindings.
-It this is necessary, extra migrations steps need to be taken for the exsing bindings in the CF's Cloud Controller.
+The `servedMssqlBindingHostname` and `servedMssqlBindingPort` properties need to be changed for every installetion. They are the hostname and port that are sent to the CF applications, and need to be accessible from the CF application network. NOTE: Do not change this value on a existing mssql broker with active bindings. If this is necessary, extra migrations steps need to be taken for the exsing bindings in the CF's Cloud Controller.
```
Comment (viovanov) [inline README.md]: typos
installetion -> installation

File: README.md [hunk 4]
```diff
@@ -3,16 +3,15 @@ A Go broker for MSSQL Service
...
+The cf-mssql-broker project implements and exposes the CF (Cloud Foundy) [Service Broker API](http://docs.cloudfoundry.org/services/api.html) to facilitate the managemnt of a single Microsoft SQL Server instance. The minimum version supported is SQL Server 2012 Express.
 
-For the provision operation of a CF service instance the broker will create a contained database (https://msdn.microsoft.com/en-us/library/ff929071.aspx) on the targeted SQL Server.
+For the provision operation of a CF service instance the broker will create a [contained database](https://msdn.microsoft.com/en-us/library/ff929071.aspx) on the targeted SQL Server.
 
 For the binding operation the broker will create an SQL Server user and a ramdomly genrated password in the contained database of the CF service instance.
 
 The broker serive does not need to have save any state, thus it can be farmed or deployed on another box without any data migration. To keep track of provisioned instances and bindings it will use the IDs from Service Broker API in the database name and in the SQL Server user name.
```
Comment (viovanov) [inline README.md]: typos:
serive -> service
to have save -> to save
ramomly -> randomly
genrated -> generated

**Comment by viovanov** [comment]
the connection string lgtm

**Comment by stefanschneider** [comment]
@viovanov  typos are fixed now. thx :)

**Comment by viovanov** [comment]
:+1:

**Pull request merged**